ACKategories/Classes/Color+Extra.swift

```diff
@@ -60,7 +60,7 @@ public extension UIColor {
 
      - parameter amount: Defines how much lighter color is returned. Should be from 0.0 to 1.0
      */
-    public func lighter(amount: CGFloat) -> UIColor {
+    public func lighter(_ amount: CGFloat) -> UIColor {
```

Comment:
fyi the conventions probably require something like `func brightened(by percent: CGFloat) -> UIColor`, but im ok with just accepting any working migration and doing the style changes later (if we decide that its worth it). We should be able to write migration rules for new versions so breaking changes arent too much of a pain.